web-animations-1/Overview.bs

```diff
@@ -1324,6 +1304,9 @@ follows:
     perform the steps according to the first matching condition from below:
 
     <div class="switch">
+    :   If |animation|'s <a>start time</a> is <a>unresolved</a> and the
+        associated <a>timeline</a> is not [=monotonically increasing=]
+    ::  Let <var>animation</var>'s <a>start time</a> be zero.
```

Comment:
(Presumably whatever we decide for the play procedure will also apply here.)

Reply:
Changed similarly to Play.